experimental/README.md

```diff
@@ -4,4 +4,4 @@
 
 | Template | Description |
 | --- | --- |
-| [`ner_spancat`](ner_spancat) | Example SpanCategorizer project using Indonesian NER |
```

Comment:
Let's remove the diff here! Just so that we won't have stray diffs in other parts of the repo! :D